tests/console/quota.pm

```diff
@@ -65,15 +75,17 @@ sub run {
     assert_script_run 'echo {1..6} |  xargs touch';
     #quota return 1 when user exceed quota limte. Line bellow accept when return is 1.
     die 'Quota should report failure' if script_run('quota') == 0;
-    assert_script_run "cd";
+    assert_script_run "cd" unless $use_templated_service;
 
     select_console 'root-console';
     #quota report
     assert_script_run "repquota /tmp/quota";
 
     #Clean configurations, stop quota, dismount disc image
-    systemctl "stop quotaon";
-    assert_script_run "cd ; umount /tmp/quota";
+    assert_script_run("systemctl stop quotaon@" . $escaped_mount) if $use_templated_service;
+    systemctl "stop quotaon" unless $use_templated_service;
+    script_retry("umount -l /tmp/quota", timeout => 180, retry => 3) if $use_templated_service;
```

Comment:
Why lazy with template ? IMO same umount should work for both cases with retry

Reply:
it didn't

Reply:
ok